### PR TITLE
Claim flow

### PR DESCRIPTION
**Pull request by sarimash**
### All Submissions

* [x] Can you post a screenshot of your changes (if applicable)?
image
Left routes to /portal, right goes to next dialog
image
Button routes to /stake, the white warning goes to last dialog
image
Left closes, right initiates the metamask confirmation to withdraw skill

### New Feature Submissions

* [x] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

Addresses https://github.com/CryptoBlades/cryptoblades/issues/110 .
The only current issues I know of is that the tax rate is hardcoded to 14% (for testing reasons) rather than retrieving from store, and the old method of claiming SKILL (clicking the unclaimed reward text) still works

### CODE REVIEW

File: frontend/src/components/Options.vue [hunk 1]
```diff
@@ -1,38 +1,107 @@
...
+<script lang="ts">
 import Events from '../events';
+import { mapActions, mapState } from 'vuex';
+import BN from 'bignumber.js';
+import Web3 from 'web3';
+import { Accessors } from 'vue/types/options';
+import { BootstrapVueIcons } from 'bootstrap-vue';
+import Vue from 'vue';
+
+Vue.use(BootstrapVueIcons);
```
Comment (seiyria) [inline frontend/src/components/Options.vue:48]: this is included globally

File: frontend/src/components/Options.vue [hunk 2]
```diff
@@ -1,38 +1,107 @@
...
+  currentWithdrawTax: number;
+  directStakeBonusPercent: number;
+}
+
+export default Vue.extend({
   created() {
     this.showGraphics = localStorage.getItem('useGraphics') === 'true';
     this.hideRewards = localStorage.getItem('hideRewards') === 'true';
     this.hideAdvanced = localStorage.getItem('hideAdvanced') === 'true';
+    this.currentWithdrawTax = 0.14;
```
Comment (seiyria) [inline frontend/src/components/Options.vue:70]: is this able to be gotten dynamically, or is it just for display purposes?

File: frontend/src/components/Options.vue [hunk 3]
```diff
@@ -56,8 +125,25 @@ export default {
...
     },
-  },
-};
+    async onClaimTokens() {
+      if(this.canClaimTokens) {
+        await this.claimTokenRewards();
+      }
+    },
+    async claimSkill(stage: number) {
+      if(stage === 0) {
```
Comment (seiyria) [inline frontend/src/components/Options.vue:134]: can this be an enum instead?

File: frontend/src/components/Options.vue (hunk 2 above)
Comment (sarimash) [reply frontend/src/components/Options.vue]: Pulling from store now, thanks

File: frontend/src/components/Options.vue (hunk 3 above)
Comment (sarimash) [reply frontend/src/components/Options.vue]: Done! It's working as is, but should I have declared the enum somewhere else or anything?

Comment (seiyria) [reply frontend/src/components/Options.vue]: since the enum is just for this page, having it only on this page is fine

**Review by raymond-h — APPROVED** [review]

**Pull request merged**